Eatery/Views/Eateries/EateryScreenViews/MealStationTableViewCell.swift

```diff
@@ -33,19 +31,58 @@ class MealStationTableViewCell: UITableViewCell {
             make.leading.trailing.equalToSuperview().inset(10)
         }
 
+        collapseTitleLabelConstraint = titleLabel.snp.prepareConstraints({ make in
+            make.height.equalTo(0)
+        }).first
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) will not be implemented")
+    }
+
+}
+
+class MealStationItemTableViewCell: UITableViewCell {
+
+    let contentLabel = UILabel()
+
+    let favoritedStatus = UIImageView()
+
+    let favoritedImage = UIImage(named: "goldStar")
+    let unfavoritedImage = UIImage(named: "unselected")
+
+    var favorited: Bool = false {
```

Comment:
```suggestion
    var favorited = false {
```

You might be able to say this. Maybe not.